wremnants/datasets/datagroups.py

```diff
@@ -64,6 +64,7 @@ def setHists(self, baseName, syst, procsToRead=None, label=None, nominalIfMissin
             if selectSignal and group[label] and "signalOp" in group and group["signalOp"]:
                 group[label] = group["signalOp"](group[label])
         # Avoid situation where the nominal is read for all processes for this syst
+        if syst == "": foundExact = True
```

Comment:
I don't understand why this is needed. Shouldn't the readHist function work fine for syst="", and then foundExact is set to true in line 55? What do you run to produce an error?

Reply:
I tested it without this line, the combine cards and input files were successfully produced, I will remove this line.

Reply:
Do you know what the issue was that caused you to run into this, though?

Reply:
I couldn't recall the details now... should probably have kept a better log. I can't reproduce now what I saw, maybe other updates fixed this bug.